l5kit/l5kit/evaluation/metrics.py

```diff
@@ -303,3 +304,22 @@ def final_displacement_error_mean(
     """
 
     return _final_displacement_error(ground_truth, pred, confidences, avails, "mean")
+
+
+def distance_to_reference_trajectory(pred_centroid: torch.Tensor, ref_traj: torch.Tensor) -> torch.Tensor:
```

Comment:
#226 works with numpy array, this one with tensors. Nothing against it if we're refactoring them later anyway

Reply:
I would like to compute the metrics using GPU, so I would use tensor whenever possible